pkg/controllers/override/overridepolicy_controller.go

```diff
@@ -176,7 +176,8 @@ func (c *Controller) enqueueFederatedObjectsForFTC(ftc *fedcorev1a1.FederatedTyp
 	logger.V(2).Info("Enqueue federated objects for FTC")
 
 	allObjects := []fedcorev1a1.GenericFederatedObject{}
-	fedObjects, err := c.fedObjectInformer.Lister().List(labels.Everything())
+	labelsSet := labels.Set{ftc.GetSourceTypeGVK().GroupVersion().String(): ftc.GetSourceTypeGVK().Kind}
+	fedObjects, err := c.fedObjectInformer.Lister().List(labels.SelectorFromSet(labelsSet))
```

Comment:
you forget to update `c.clusterFedObjectInformer.Lister().List(labelselector)`,  or we should list like this:
``` go
if ftc.namespaced(){
fedObjects, err := c.fedObjectInformer.Lister().List(labels.SelectorFromSet(labelsSet))
} else{
fedObjects, err := c.clusterFedObjectInformer.Lister().List(labels.SelectorFromSet(labelsSet))
}

Reply:
The code has been updated.